projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -48,8 +49,8 @@ export function verifyTicketListingPageVisit(){
   cy.get('cx-customer-ticketing-list').should('exist');
 }
 
-export function clickFirstTicketFromTicketListing(){
-  cy.get(`#ticketing-list-table tbody tr:nth-child(${FIRST_TICKET_ROW_INDEX}) .cx-ticketing-list-data:nth-child(${FIRST_TICKET_COLUMN_INDEX}) a.cx-ticketing-list-value`).click();
+export function clickTicketFromTicketListing(ticketRowIndex: number){
+  cy.get(`#ticketing-list-table tbody tr:nth-child(${ticketRowIndex}) .cx-ticketing-list-data:nth-child(${TICKET_ID_COLUMN_INDEX}) a.cx-ticketing-list-value`).click();
```

Comment:
I think `cy.get('cx-customer-ticketing-list').get('tbody').get('tr').eq(ticketRowIndex).click();` might be simpler. Used [here](https://github.com/SAP/spartacus/blob/48fb4363e7a3f172d852c76e39d01ac6098b8b7f/projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts#L124)